lib/bamboo/adapters/send_grid_helper.ex

```diff
@@ -135,6 +136,15 @@ defmodule Bamboo.SendGridHelper do
     raise "expected the asm_group_id parameter to be an integer, got #{asm_group_id}"
   end
 
+  def with_bypass_list_management(email, enabled) when is_boolean(enabled) do
```

Comment:
Thoughts on adding some small documentation with an example here to demonstrate how to use this?